bus/bus.go

```diff
@@ -958,6 +959,25 @@ func (b *bus) objectsHandlerGET(jc jape.Context) {
 }
 
 func (b *bus) objectEntriesHandlerGET(jc jape.Context, path string) {
+	bucket := api.DefaultBucketName
+	if jc.DecodeForm("bucket", &bucket) != nil {
+		return
+	}
+
+	var prefix string
+	if jc.DecodeForm("prefix", &prefix) != nil {
+		return
+	} else if unescaped, err := url.PathUnescape(prefix); err == nil {
```

Comment:
Do we really need to unescape here? Shouldn't the router unescape the query params implicitly?

Reply:
removed it for now since my tests pass without it now but I found that passing prefix including a slash ended up being unescaped when they were passed to the store